Gemfile

```diff
@@ -69,6 +69,10 @@ gem 'skylight'
 gem 'sidekiq'
 gem 'dalli'
 
+# admin functionality
+gem "datagrid"
```

Comment:
Can we remove `datagrid` until we identify an explicit need on the frontend? I prefer to be very pessimistic on dependencies when possible.

Reply:
Oh yep, Leftover from when I was experimenting with using it.